From 1170d1ba6d8d033c4761bc7ea14299f5fc6f37c0 Mon Sep 17 00:00:00 2001
From: "Kevin F. Haggerty" <haggertk@lineageos.org>
Date: Thu, 15 Nov 2018 21:03:55 -0700
Subject: [PATCH 12/19] msm8974-common: sepolicy: Resolve surfaceflinger
 denials

* avc: denied { read } for name="mdp" dev="sysfs" ino=12906
  scontext=u:r:surfaceflinger:s0 tcontext=u:object_r:sysfs_graphics:s0
  tclass=lnk_file permissive=0

Change-Id: Idf97698291e769b54a2338a7afc0d940d523b605
---
 sepolicy/common/surfaceflinger.te | 1 +
 1 file changed, 1 insertion(+)
 create mode 100644 sepolicy/common/surfaceflinger.te

diff --git a/sepolicy/common/surfaceflinger.te b/sepolicy/common/surfaceflinger.te
new file mode 100644
index 0000000..0ab14ae
--- /dev/null
+++ b/sepolicy/common/surfaceflinger.te
@@ -0,0 +1 @@
+allow surfaceflinger sysfs_graphics:lnk_file read;
-- 
2.17.1

